### PR TITLE
Quick fix and improvement on team#update

### PR DESCRIPTION
**Pull request by vitoravelino**
Also added alert notification and removed unused variables.

Signed-off-by: Vítor Avelino <contact@vitoravelino.me>

### CODE REVIEW

File: app/assets/javascripts/teams.js [hunk 1]
```diff
@@ -50,7 +51,7 @@ jQuery(function ($) {
     });
   });
 
-  $('body').on('click', '.btn-edit-webhook', function (_event) {
+  $('body').on('click', '.btn-edit-webhook', function (event) {
     if ($(this).hasClass('button_edit_webhook')) {
       $('.webhook_information').toggle();
       $('#update_webhook_' + event.currentTarget.value).toggle();
```
Comment (vitoravelino) [inline app/assets/javascripts/teams.js]: `event` was being used instead of `_event`.

File: app/assets/javascripts/teams.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 jQuery(function ($) {
-  $('#add_team_user_btn').on('click', function (_event) {
+  $('#add_team_user_btn').on('click', function () {
```
Comment (mssola) [inline app/assets/javascripts/teams.js]: Is this parameter really optional then ? I wrote `_event` to state "jQuery wants to pass a parameter but we don't use it". If so, I think that we can strip most of the parameters that start with "_" 😄

File: app/assets/javascripts/teams.js (hunk 1 above)
Comment (mssola) [inline app/assets/javascripts/teams.js]: Yeah, good catch 👍  Note that we write variables starting with _ when we have to have a function parameter but we don't use it (see my comment above). And it looks like I didn't see this usage.

**Comment by mssola** [comment]
Thanks a bunch 👏

File: app/assets/javascripts/teams.js (hunk 2 above)
Comment (vitoravelino) [inline app/assets/javascripts/teams.js]: Yes, it's optional. By default there's no check on how functions are referenced/called. JS sucks on those kind of stuff and that's one of the reasons why TypeScript is getting more and more popular and being used in bigger projects.

Just saw that you've configured ESLint to ignore unused variables starting with `_`. I guess you can remove that and let ESLint do the job of checking. 

I know that eventually there will be a case like:

```js
(not_used, used) => {
  // ...
}
```

and you can use `_not_used` without needing to configure ESLint to not complain about it. ESLint is smart enough to know that `used` is used and the first parameter is necessary to be declared even if not used. So tou don't need to configure to ignore variables starting with `_`.

On the other hand, keeping the `_var` indicates that the function receives that parameter but it's not being used in it. If an app gets too big and lacks documentation, having that artifact would help a lot. In this case, it's a well documented and know scenario of listening to a dom event through jQuery. I don't think it's necessary, but it's your call to keep it or not.

I also realized that `event` is a reserved keyword and that's another reason why ESLint didn't catch the erro of `event` not being defined. For events I'm comfortable with `e` or `evt`.

What do you think?

Comment (mssola) [inline app/assets/javascripts/teams.js]: Totally agree with you. Thanks for the explanation ! 👍

**Pull request merged**